### DISCUSSION TITLE
/ontime/message/external/visible true | false

### DISCUSSION THREAD

**Discussion by craigreilly** [cpvalente/ontime#1030] [Ideas]
Any chance this could be a toggle instead of TRUE or FALSE only? Or get FEEDBACK on the status?

**Comment by alex-Arc** [comment]
I've been looking at getting some feedback working a couple of times, right now we are blocked on the OSC library
https://github.com/MylesBorins/node-osc/issues/116

A toggle option could also be something to consider, how do you imagen the osc message would look?
